### ISSUE TITLE
Live-reload fails if --config-file option is used

### ISSUE THREAD

**Issue by seelensonne** [mkdocs/mkdocs#741]
Test Case:

```bash
$ mkdocs --version # mkdocs, version 0.14.0
$ mkdocs serve --config-file foo.yml --livereload --verbose
$ touch docs/index.md
```

Expected:

```
INFO    -  Building documentation...
DEBUG   -  Loading configuration file: foo.yml
[...]
```

Actual:

```
INFO    -  Building documentation...
DEBUG   -  Loading configuration file: <closed file 'foo.yml', mode 'rb' at 0x1063d6660>
[E 151020 13:25:23 ioloop:612] Exception in callback <bound method type.poll_tasks of <class 'livereload.handlers.LiveReloadHandler'>>
    Traceback (most recent call last):
      File "/usr/local/lib/python2.7/site-packages/tornado/ioloop.py", line 1020, in _run
        return self.callback()
      File "/usr/local/lib/python2.7/site-packages/livereload/handlers.py", line 65, in poll_tasks
        filepath, delay = cls.watcher.examine()
      File "/usr/local/lib/python2.7/site-packages/livereload/watcher.py", line 72, in examine
        func and func()
      File "/usr/local/lib/python2.7/site-packages/mkdocs/serve.py", line 71, in builder
        theme=theme,
      File "/usr/local/lib/python2.7/site-packages/mkdocs/config/base.py", line 143, in load_config
        cfg.load_file(config_file)
      File "/usr/local/lib/python2.7/site-packages/mkdocs/config/base.py", line 96, in load_file
        return self.load_dict(utils.yaml_load(config_file))
      File "/usr/local/lib/python2.7/site-packages/mkdocs/utils/__init__.py", line 58, in yaml_load
        return yaml.load(source, Loader)
      File "/usr/local/lib/python2.7/site-packages/yaml/__init__.py", line 69, in load
        loader = Loader(stream)
      File "/usr/local/lib/python2.7/site-packages/yaml/loader.py", line 34, in __init__
        Reader.__init__(self, stream)
      File "/usr/local/lib/python2.7/site-packages/yaml/reader.py", line 85, in __init__
        self.determine_encoding()
      File "/usr/local/lib/python2.7/site-packages/yaml/reader.py", line 124, in determine_encoding
        self.update_raw()
      File "/usr/local/lib/python2.7/site-packages/yaml/reader.py", line 178, in update_raw
        data = self.stream.read(size)
    ValueError: I/O operation on closed file
```

**Comment by d0ugal** [comment]
This is essentially a duplicate of #735, but it is good to have a bug and not just a PR so I'll keep this open.

**Comment by waylan** [comment]
Fixed in #883.

**Comment by daviestar** [comment]
This patch hasn't been released as a tag, just hit the same issue.  Cheers.
